.github/workflows/update-engines-version.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - uses: pnpm/action-setup@v2.2.2
         with:
-          version: 6
+          version: 7
 
       - uses: actions/setup-node@v3
         with:
```

Comment:
We need to check and update this file `.github/workflows/update-engines-version.yml`

Reply:
Done.